components/NoteTags/NoteTags.js

```diff
@@ -20,7 +23,19 @@ const NoteTags = ({ classes, ...rest }) => {
       {event?.tags
         ?.filter((tag) => tag[0] == "t")
         .map((tag, index) => (
-          <Chip radius="md" key={index} className={classes.tag} checked={false}>
+          <Chip
+            radius="md"
+            key={index}
+            className={classes.tag}
+            checked={false}
+            onClick={() =>
+              router.push({
+                pathname: Route.Tags,
+                query: { tag: tag[1] },
+              })
+            }
+            styles={{ label: { cursor: "pointer !important" } }}
```

Comment:
don't need this. just get rid of cursor: "default" in the tag class in Note.styles.js